packages/core/src/external/aws/s3.ts

```diff
@@ -23,6 +24,17 @@ export const createS3FileName = (cxId: string, patientId: string, fileName: stri
   return `${cxId}/${patientId}/${cxId}_${patientId}_${fileName}`;
 };
 
+export const createMRSummaryFileName = (
+  cxId: string,
+  patientId: string,
+  extension: "pdf" | "html" | "json"
+): string => {
+  if (extension === "pdf") {
+    return createS3FileName(cxId, patientId, `${MEDICAL_RECORD_KEY}.html.pdf`);
+  }
+  return createS3FileName(cxId, patientId, `${MEDICAL_RECORD_KEY}.${extension}`);
+};
```

Comment:
I don't think we should mix metriport logic with S3 code. This file should be exclusive to S3 operations, i.e., `parseS3FileName()` should not be here either, IMO.